includes/internal/emails/class-email-customization.php

```diff
@@ -55,16 +55,32 @@ class Email_Customization {
 	 */
 	private $settings_tab;
 
+	/**
+	 * Sensei_Email_Sender instance.
+	 *
+	 * @var Email_Sender
+	 */
+	private $email_sender;
+
+	/**
+	 * Sensei_Email_Generator instance.
+	 *
+	 * @var Sensei_Email_Generator
+	 */
+	private $email_generator;
+
 	/**
 	 * Email_Customization constructor.
 	 *
 	 * Prevents other instances from being created outside of `self::instance()`.
 	 */
 	private function __construct() {
-		$this->post_type     = new Email_Post_Type();
-		$this->settings_menu = new Settings_Menu();
-		$this->settings_tab  = new Email_Settings_Tab();
-		$this->blocks        = new Email_Blocks();
+		$this->post_type       = new Email_Post_Type();
+		$this->settings_menu   = new Settings_Menu();
+		$this->settings_tab    = new Email_Settings_Tab();
+		$this->blocks          = new Email_Blocks();
+		$this->email_sender    = Email_Sender::instance();
```

Comment:
Is there some reason to have it is different from the others?

Reply:
Changed it here, thanks https://github.com/Automattic/sensei/pull/6533/commits/69a0cac2d3a499d1c5549e43e9228a8e3ebb259b